### ISSUE TITLE
Parsing error

### ISSUE THREAD

**Issue by gaetanofan** [mit-cml/aiatools#4]
Hi, It seems that the tools is not working anymore with the current version of Mit App Inventor 2.
For my understandings the issue arises when parsing the .bky file.
I have an aia file containing a .bky file with the following information
```
...
<yacodeblocks ya-version="213" language-version="34"></yacodeblocks>
...
```

**Comment by halatmit** [comment]
Please attach or link to the aia file so we can see what the error is.

**Comment by gaetanofan** [comment]
Please find attached the aia file.

[temp.zip](https://github.com/mit-cml/aiatools/files/7783723/temp.zip)

**Comment by halatmit** [comment]
The zip archive you attached is empty.  Also, don't just send the aia file.  Also say what we should do to stimulate the error: What should one expect to happen and what actually happens.
Thanks.

**Comment by gaetanofan** [comment]
Sorry for the inconvenience. The code used for reproducing the error is just a modification of the sample code found in the package. I attach the aia files, the sample code and the log files of stderr. Hope it helps, thanks.

[temp.aia.gz](https://github.com/mit-cml/aiatools/files/7787848/temp.aia.gz)
[mysamples.py.gz](https://github.com/mit-cml/aiatools/files/7787870/mysamples.py.gz)
[log.txt](https://github.com/mit-cml/aiatools/files/7787871/log.txt)

**Comment by halatmit** [comment]
This might be a parsing error in aiatools

**Comment by ewpatton** [comment]
The project in question includes the new helper blocks added by @BeksOmega. I've added a new commit caf1c9455c8ea8f1c57e9bcba6239eeafea11327 that should address the new block category and published version 0.1.7.

**Comment by gaetanofan** [comment]
The commit fixes the bug for my test cases. Thank you for addressing it.
